core/frontend/src/PerModelCategoryVisibility.ts

```diff
@@ -56,6 +59,13 @@ export namespace PerModelCategoryVisibility {
     addOverrides(fs: FeatureSymbology.Overrides, ovrs: Id64.Uint32Map<Id64.Uint32Set>): void;
   }
 
+  /** Props to describe per model category visibility. */
+  export interface Props {
```

Comment:
Properties of `@public` interface require documentation. Add `@see` links to where this interface is useful.

Reply:
added a @see link